include/CLE.h

```diff
@@ -83,6 +83,8 @@ class CLE
     void SmallerOrEqual(Buffer&, Buffer&, Buffer&);
     void SmallerConstant(Buffer&, Buffer&, float);
     void SmallerOrEqualConstant(Buffer&, Buffer&, float);
+    void LabelConnectComponent(Buffer&, Buffer&);
```

Comment:
If you name it "ConnectedComponentsLabelingBox", then the code generator would work with it.